src/Uno.UI/UI/Xaml/Controls/ScrollBar/ScrollBar.uno.cs

```diff
@@ -42,9 +44,17 @@ private static void DetachEvents(object snd, RoutedEventArgs args) // OnUnloaded
 
 	internal bool HasFixedVisualStates()
 	{
+#if __SKIA__
+		return false;
```

Comment:
Should it be tested somehow?

Reply:
No need, the fixed visual states were completely disabled on all targets except WASM in previous ScrollBar PR, so this is essentially just no-op